src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
GitExtensions is showing me the `u` changed, but I see no difference in Visual Studio. What's up with these two changes?